init.coffee

```diff
@@ -13,11 +13,30 @@ exports.configure = [
     message: 'Do you need netlify hosting?'
   },
   {
+    type: 'list'
+    name: 'css'
+    message: 'What would you like to use as your CSS Preprocessor?'
+    choices: ['postcss', 'stylus']
```

Comment:
well shit, let's go with that. wouldn't be tough to add postcss/stylus separately if we ever encounter a problem